rollup.config.js

```diff
@@ -5,14 +5,15 @@ import json from 'rollup-plugin-json'
 import commonjs from 'rollup-plugin-commonjs'
 import resolve from 'rollup-plugin-node-resolve'
 import { terser } from 'rollup-plugin-terser'
+import license from 'rollup-plugin-license'
 
 const bundle = (input, output, options = {}) => ({
   input: `src/build/${input}`,
 
   output: {
     name: 'uploadcare',
     format: 'umd',
-    file: `dist/${output}`,
+    file: `${output}`,
```

Comment:
why not `/dist/`?

Reply:
Reason is backward compatibility. In `3.7.9` you can `require('uploadcare-widget/uploadcare-widget.lang.en')`, if we place all bundles in `dist` folder this functionality breaks